src/databaseInterface/src/main/java/org/vaccineimpact/orderlyweb/db/DefaultSchema.java

```diff
@@ -68,7 +69,7 @@
 @SuppressWarnings({ "all", "unchecked", "rawtypes" })
 public class DefaultSchema extends SchemaImpl {
 
-    private static final long serialVersionUID = 1999477293;
+    private static final long serialVersionUID = 1282328126;
```

Comment:
I am sure this is right but am quite curious about this and how it is derived - or is this whole file autogenerated?

Reply:
Yup, this is all auto-generated by Jooq